src/main/java/uk/gov/hmcts/darts/notification/service/impl/NotificationServiceImpl.java

```diff
@@ -106,6 +106,7 @@ public void sendNotificationToGovNotify() {
                     govNotifyRequest,
                     e.getMessage()
                 );
+                log.error("notification failed to send");
```

Comment:
this is probably not needed, as there is an error being logged on the line above.